### PR TITLE
COMMERCE-4808 When creating a second Minium site, the base price lists are created without a catalog name

### PR DESCRIPTION
**Pull request by riccardo-alberti**
@marco-leo i had to restore the getGroupId otherwise it will return a NoSuchGroup exception

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java [hunk 1]
```diff
@@ -145,13 +145,15 @@ public CommerceCatalog deleteCommerceCatalog(
...
+		long groupId = commerceCatalog.getGroupId();
+
 		// Commerce catalog
 
 		commerceCatalogPersistence.remove(commerceCatalog);
 
 		// Group
 
-		groupLocalService.deleteGroup(commerceCatalog.getGroupId());
+		groupLocalService.deleteGroup(groupId);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java:156]: @riccardo-alberti I think itd be more clear if we just moved
```
// Group

groupLocalService.deleteGroup(commerceCatalog.getGroupId());
```
before
```
// Commerce catalog

commerceCatalogPersistence.remove(commerceCatalog);
```

if that solves NoSuchGroup exception

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java (hunk 1 above)
Comment (riccardo-alberti) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java:156]: @dsitu if the persistence is removed after the group is deleted the model listener on the catalog will fail with NoSuchGroupException

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**